### PR TITLE
fix logrotate for *arr packages

### PR DESCRIPTION
**Pull request by PF4Public**

### CODE REVIEW

File: www-apps/prowlarr/files/prowlarr.logrotate [hunk 1]
```diff
@@ -1,4 +1,4 @@
-/var/lib/prowlarr/.config/prowlarr/logs/*.log{
+/var/lib/prowlarr/.config/Prowlarr/logs/*.txt{
```
Comment (thesamesam) [inline www-apps/prowlarr/files/prowlarr.logrotate:1]: Please revbump for these fixes (git mv is fine, as it's all binary anyway).

Comment (PF4Public) [reply www-apps/prowlarr/files/prowlarr.logrotate:1]: @thesamesam you mean revbumping ebuilds themselves and not logrotate files, right?

Comment (thesamesam) [reply www-apps/prowlarr/files/prowlarr.logrotate:1]: yes, sorry :)

**Comment by PF4Public** [comment]
@thesamesam Done

**Pull request closed without merging**